scripts/create-example-tests.js

```diff
@@ -1144,9 +1147,24 @@ function validateTest(testParsed, data, { addTestError = () => {} } = {}) {
  * @param {Queryable<AriaATParsed.Key>} data.key
  * @param {Queryable<{name: string, path: string}>} data.example
  * @param {Queryable<{at: string, mode: string, render: function({key: *}): string}>} data.modeInstructionTemplate
+ * @param {string} testPlanDirectory
  * @returns {AriaATFile.CollectedTest}
  */
-function collectTestData({ test, command, key, example, modeInstructionTemplate }) {
+function collectTestData(
+  { test, command, key, example, modeInstructionTemplate },
+  testPlanDirectory
```

Comment:
I think we should keep file path information outside of these functions (`collectTestData`, `validateTest`, `parseCommandCSVRow`, etc). We can add path information in the main function as part of defining `addTestError` and `addCommandError` for example.